### ISSUE TITLE
Waterfall navigation doesn't scroll + I found CDN material.min.js is running errors

### ISSUE THREAD

**Issue by missamii** [google/material-design-lite#5021]
## possible bugs?

> Latest version via CDN. Version 1.3.0
> I'm using Chrome Version 56.0.2924.76 (64-bit)
> macOS Sierra version 10.12.2 (16C67)
>The behavior I expected was that of the demo on MDL site. Where when I scrolled the nav bar would shrink. I tried added content, but it only stretches the page.
> It's sticking with the top of page and when you scroll down it scrolls away. I believe it should be acting as sticky header/nav bar

> When I opened my console log, I found the following:

I'm using this:
<script defer src="https://code.getmdl.io/1.3.0/material.min.js"></script>

And in my console, it's giving me the following errors
```
layout.js:309 Uncaught TypeError: Cannot read property 'addEventListener' of undefined
    at f.init (layout.js:309)
    at new f (layout.js:28)
    at a (mdlComponentHandler.js:241)
    at n (mdlComponentHandler.js:192)
    at Object._ [as upgradeAllRegistered] (mdlComponentHandler.js:360)
    at mdlComponentHandler.js:492
```

**Comment by shadrackkip** [comment]
Did you find a solution to this problem? coz am experiencing the same problem.

**Comment by missamii** [comment]
No, I never did. I ended up moving on to something else. My client needed something that would work. And this wasn't doing it. Maybe I'll go back and play with it again. See if ideas come to mind.
